src/test/java/com/websdkAutomation/HomePageTest.java

```diff
@@ -28,16 +22,13 @@
 	public void setUp() 
 	{
 		driver=Fixtures.setupFixture("Home page test class setup initiated.");
-		String BaseWindowHandle = Browser.getSwitchto().getBasewindowHandle(driver);
+		/*String BaseWindowHandle = */
```

Comment:
we are  using this "String BaseWindowHandle" variable only once. hence removed variable declaration for avoiding grabage collection